core/dbt/events/functions.py

```diff
@@ -0,0 +1,213 @@
+
+from colorama import Style
+import dbt.events.functions as this  # don't worry I hate it too.
```

Comment:
Why is this needed?

Reply:
yeah I just tried deleting all the `this` calls, and flake8 gives me `F841 local variable 'format_json' is assigned to but never used` 🤷‍♂️